Scripts/StixParser/StixParser.py

```diff
@@ -184,6 +184,21 @@ def extract_indicators(data):
         dict: containing all indicators data
 
     """
+    def get_indicators(indicator):
```

Comment:
take it out and add test for it

Reply:
problem with taking it out. anyway, it'll be used only in this function. you can use this function with sending input to the parent function (which only checks if we got dict or list